evennia/contrib/utils/debugpy/README.md

```diff
@@ -0,0 +1,157 @@
+# In-game debugpy command
+
+This registers an in-game command `debugpy` which starts the debugpy debugger and listens on port 5678.
+For now this is only available for Visual Studio Code (VS Code).
+
+If you are a JetBrains PyCharm user and would like to use this, make some noise at:
+https://youtrack.jetbrains.com/issue/PY-63403/Support-debugpy
+
+
+Credit for this goes to Moony on the Evennia Discord getting-help channel, thx Moony!
+
+0xDEADFED5 simply tied a pretty bow around it and stuck it here for everybody else.
+
+
+
+## Dependencies
+
+This requires VS Code and debugpy, so make sure you're using VS Code.
+
+From the venv where you installed Evennia run:
+
+`pip install debugpy`
+
+## Enable the command in Evennia
```

Comment:
Same here. See the the guidelines and structure the README on the standard form.